LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -39,7 +39,8 @@ @implementation UIApplication (window)
 + (UIWindow *)currentWindow {
     UIWindow *window;
 #ifdef LTH_IS_APP_EXTENSION
-    window = [UIApplication sharedApplication].keyWindow;
+    NSPredicate *isKeyWindow = [NSPredicate predicateWithFormat:@"isKeyWindow == YES"];
+    window = [UIApplication.sharedApplication.windows filteredArrayUsingPredicate:isKeyWindow].firstObject;
```

Comment:
```@property(class, nonatomic, readonly) UIApplication *sharedApplication NS_EXTENSION_UNAVAILABLE_IOS("Use view controller based solutions where appropriate instead.");
```
sharedApplication is not available on extensions?

Reply:
Thanks! I will have updated the code. Please have a look and let me know your thoughts. Thanks!

Reply:
Nice, thanks